project/MiMa.scala

```diff
@@ -1152,10 +1152,27 @@ object MiMa extends AutoPlugin {
       //  * this list ends with the latest released version number
       //  * is kept in sync between release-2.4 and master branch
     )
+    
+    val Release25Filters = Seq(
+      "2.5.0" -> Seq(
+          
+          // #22759 LMDB files
+          ProblemFilters.exclude[DirectMissingMethodProblem]("akka.cluster.ddata.LmdbDurableStore.valueBuffer_="),
+          ProblemFilters.exclude[DirectMissingMethodProblem]("akka.cluster.ddata.LmdbDurableStore.db"),
+          ProblemFilters.exclude[DirectMissingMethodProblem]("akka.cluster.ddata.LmdbDurableStore.keyBuffer"),
+          ProblemFilters.exclude[DirectMissingMethodProblem]("akka.cluster.ddata.LmdbDurableStore.env"),
+          ProblemFilters.exclude[DirectMissingMethodProblem]("akka.cluster.ddata.LmdbDurableStore.valueBuffer"),
+          
+          ProblemFilters.exclude[InheritedNewAbstractMethodProblem]("akka.stream.Graph.traversalBuilder"),
+          ProblemFilters.exclude[InheritedNewAbstractMethodProblem]("akka.stream.Graph.named"),
+          ProblemFilters.exclude[InheritedNewAbstractMethodProblem]("akka.stream.Graph.addAttributes"),
+          ProblemFilters.exclude[InheritedNewAbstractMethodProblem]("akka.stream.Graph.async")
```

Comment:
@2m I think the `akka.stream.Graph` incompatibilities reported here were filtered by the `bcIssuesBetween24and25` when we released 2.5.0. For example:

```
ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.Graph.traversalBuilder"),
```

I'm pretty confused about that the log message says "checking against com.typesafe.akka:akka-stream_2.11:2.4.12", and why it was a `ReversedMissingMethodProblem` and now  `InheritedNewAbstractMethodProblem`

I think all is good though.